Description: Misc fixups to make rocksdb work on anything other than x86
Author: James Page <james.page@ubuntu.com>
Forwarded: no

--- a/src/rocksdb/Makefile
+++ b/src/rocksdb/Makefile
@@ -84,7 +84,9 @@ endif
 # compile with -O2 if debug level is not 2
 ifneq ($(DEBUG_LEVEL), 2)
 OPT += -O2 -fno-omit-frame-pointer
-ifneq ($(MACHINE),ppc64) # ppc64 doesn't support -momit-leaf-frame-pointer
+# Skip for archs that don't support -momit-leaf-frame-pointer
+no_omit_leaf_frame_pointer = ppc ppc64 ppc64le armv7l s390x
+ifeq (,$(filter $(MACHINE), $(no_omit_leaf_frame_pointer)))
 OPT += -momit-leaf-frame-pointer
 endif
 endif
